### DISCUSSION TITLE
request: Add Dasel

### DISCUSSION THREAD

**Discussion by ghostsquad** [asdf-community/.github#11] [General]
https://github.com/TomWright/dasel

Related issue: https://github.com/TomWright/dasel/issues/78

**Comment by smorimoto** [comment]
It seems very easy to support, but I don't currently have enough bandwidth to do it.

**Comment by TomWright** [comment]
I'd be happy if dasel were added to asdf. I would like to request maintainer status if this goes through though so I can ensure any required changes are made.

**Reply by smorimoto** [reply]
We manage GitHub state with Terraform to make our process open as well. In other words, you can create a repository and add yourself as a collaborator of that repository by opening a PR here: https://github.com/asdf-community/infrastructure

**Comment by ghostsquad** [comment]
https://github.com/asdf-community/infrastructure/pull/1

**Reply by smorimoto** [reply]
@ghostsquad @TomWright Since I merged it, a repository has been created and you are now invited there as collaborators!

**Reply by smorimoto** [reply]
UPDATE: I actually created a team and added you there.

**Reply by TomWright** [reply]
Thanks!
